### PR TITLE
wip: compare status of origin and CloudFormation

### PR DESCRIPTION
**Pull request by aartaria**

### CODE REVIEW

File: scripts/ci-deploy.sh [hunk 1]
```diff
@@ -52,6 +52,33 @@ for module in "${MODULES[@]}"; do
...
+    # check that CloudFormation stacks are up to date with the origin
+    if [[ $ENV == "dev" ]]; then
+      origin_branch="develop"
+    else
+      origin_branch="main"
+    fi
+    software_component=$(cat playbook.yml | grep 'software_component' | awk '{ print $2}' | sed "s/\"//g" | sed "s/'//g")
+    git_commits=($(git rev-parse origin/$origin_branch))
+    for stack in $(aws cloudformation list-stacks --output text --query 'StackSummaries[?contains(StackName, `'$software_component'`)].[StackName]'); do
+    git_commit=$(aws cloudformation describe-stacks --stack-name $stack --query 'Stacks[0].Tags[?contains(Key, `GitCommit`)][].Value' | jq -r '.[0]')
```
Comment (zhaow-de) [inline scripts/ci-deploy.sh:65]: indent

Comment (zhaow-de) [inline scripts/ci-deploy.sh:65]: prepare for the edge case, when `aws` returns nothing, `jq` fails.

**Review by zhaow-de — APPROVED** [review]
LGTM. To make it more applicable to more scenarios, especially after the rebase, more tests on the edge cases would be helpful.

**Comment by aartaria** [comment]
not relevant
